src/main/java/com/blog/web/models/Article.java

```diff
@@ -0,0 +1,111 @@
+package com.blog.web.models;
+
+import com.blog.web.dto.ArticleDto;
+import jakarta.persistence.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDateTime;
+
+@Entity
+public class Article {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
indicate final for ones that are to ensure immutability.... also why non-primative